### PR TITLE
Try to find interface of class when apply mapper (content resolver)

### PR DESCRIPTION
**Pull request by nikitin-da**
Copy paste to content resolver module 
@artem-zinnatullin PTAL please

### CODE REVIEW

File: storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/impl/DefaultStorIOContentResolver.java [hunk 1]
```diff
@@ -264,6 +279,25 @@ protected InternalImpl(@Nullable Map<Class<?>, ContentResolverTypeMapping<?>> ty
             }
         }
 
+        @Nullable
+        @SuppressWarnings("unchecked")
+        private <T> ContentResolverTypeMapping<T> findMappingClassByInterface(final Class<T> type) {
```
Comment (artem-zinnatullin) [inline storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/impl/DefaultStorIOContentResolver.java]: nullability of `type`

**Comment by artem-zinnatullin** [comment]
:+1:

**Comment by nikitin-da** [comment]
fixed!

**Comment by artem-zinnatullin** [comment]
Thanks :)

**Pull request merged**